LocalSettings.php

```diff
@@ -3085,6 +3085,15 @@
 	],
 
 	// PageForms
+	'wgPageFormsRenameEditTabs' => [
+		'default' => false,
+	],
+	'wgPageFormsRenameMainEditTab' => [
+		'default' => false,
+	],
+	'wgPageFormsSimpleUpload' => [
+		'default' => false,
+	],
 	'wgPageFormsLinkAllRedLinksToForms' => [
 		'default' => false,
 		'frontierrpgwiki' => true,
```

Comment:
please remove existing overrides.

Reply:
Could I ask wheater it is possible without contacting [frontierrpgwiki](https://frontierrpg.miraheze.org/)? Or would someone port the config into the database?

Reply:
We can add the config when deploying

Reply:
https://github.com/miraheze/ManageWiki/blob/master/maintenance/populateWikiSettings.php

Reply:
Cool, done. Thank you.